samples/meetings-token-app/csharp/ClientApp/src/Containers/MeetingTokenApp.js

```diff
@@ -5,9 +5,6 @@ import MeetingServiceProvider from '../Context/MeetingServiceProvider';
 import TeamsContextProvider from '../Context/TeamsContextProvider';
 
 class MeetingTokenApp extends Component {
-    constructor(props) {
-        super(props);
-    }
```

Comment:
Why we have removed constructor?

Reply:
I am warning this - **useless Constructor**
solution - 
A "useless constructor" is one that the linter is warning you can safely remove from the code, because it doesn't accomplish anything - if all you have is a super call (with the same argument the class is created with), the constructor doesn't do anything useful, because classes will already call super automatically, if the constructor isn't given: